.azuredevops/components/rocDecode.yml

```diff
@@ -25,23 +37,46 @@ jobs:
   - template: /.azuredevops/variables-global.yml
   pool:
     vmImage: ${{ variables.BASE_BUILD_POOL }}
-  container:
-    image: ${{ variables.DOCKER_IMAGE_NAME }}:${{ variables.LATEST_DOCKER_VERSION }}
   workspace:
     clean: all
   steps:
+  # Since mesa-amdgpu-multimedia-devel is not directly available from apt, register it
+  - task: Bash@3
+    displayName: 'Register ROCm packages'
+    inputs:
+      targetType: inline
+      script: |
+        sudo mkdir --parents --mode=0755 /etc/apt/keyrings
+        wget https://repo.radeon.com/rocm/rocm.gpg.key -O - | gpg --dearmor | sudo tee /etc/apt/keyrings/rocm.gpg > /dev/null
+        echo "deb [arch=amd64 signed-by=/etc/apt/keyrings/rocm.gpg] https://repo.radeon.com/amdgpu/6.1.1/ubuntu jammy main" | sudo tee /etc/apt/sources.list.d/amdgpu.list
```

Comment:
Here it's using https://repo.radeon.com/amdgpu/6.1.1/ubuntu-jammy-main. Concerned about ROCm version updates in the future. Shall we replace 6.1.1 with a changeable variable from global-variables?

Reply:
You can use $(LATEST_RELEASE_TAG) in place of 6.1.1 in the string and it should be fine. It will point to 6.1.0 in this scenario, which should still work.

We can reach out to see how we can pull in packages more up to date.

Reply:
Looks like 6.1.0 does not exist. Tried to switch to 6.1.0 and it outputs the error:

E: The repository 'https://repo.radeon.com/amdgpu/6.1.0/ubuntu jammy Release' does not have a Release file.

Reply:
All these keyring codes are for installing mesa-amdgpu-va-drivers. rocDecode can successfully build without it, but this PR adds it: https://github.com/ROCm/ROCm/commit/3112f97fe8b41cad1bc6d4e569edf17935c578e9

Not sure if we need it.